style.css

```diff
@@ -158,3 +158,175 @@ body {
   top: 9.35px;
   margin: 0 12px;
 }
+
+/* Work Section */
+.work-container {
+  display: grid;
+  grid-template-columns: 1;
+  grid-template-rows: 2;
+  width: 100%;
+  padding: 0 20px 0 20px;
+  margin-top: 24px 0 12px 0;
+  background: url(images/portfolio\ background\ shapes\ mobile@2x.png) top center no-repeat;
+  top: 1175px;
+  color: #dcdce8;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.work-tittle h1 {
+  display: flex;
+  justify-content: space-between;
+  color: #9490ff;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  list-style: 52px;
+  padding-top: 32px;
+  margin-top: 24px;
+}
+
+.projects-seection .post-tittle {
+  color: #f5f5ff;
+}
+
+.projects-seection ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  list-style: none;
+}
+
+.projects-seection li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  border-right: 1px solid #555;
+
+  /* padding: 8px 0; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.